grants/milestone_deliveries/season_2/package_manager/package-manager-milestone-1.md

```diff
@@ -0,0 +1,47 @@
+# Milestone Delivery :mailbox: - Milestone 1
```

Comment:
Nit: 

Thinking out loud as I think of organizing / review workflow...

Maybe we should name this current file like:
```
milestone-1-delivery.md
```

**Reasoning:**
The current `package-manager-milestone-1.md` name overlaps enough with `milestone1-part1-details.md`, such that one has to really dig into the notes to understand the difference between these files

I'd argue the hierarchy of high-level info -> low level details is:
1. Milestone delivery (current file)
2. The `*-details.md` files 

`package-manager-*` not needed as it's just above in the directory

Reply:
Good idea! I renamed the files so that they are less confusing, and there is less overlap in their names.